modules/core/src/v2/trading/tradingAccount.ts

```diff
@@ -2,6 +2,7 @@
  * @prettier
  */
 import * as Bluebird from 'bluebird';
+import BigNumber from 'bignumber.js'
```

Comment:
`import { BigNumber }`, it's better to not use the default export.

See https://basarat.gitbooks.io/typescript/docs/tips/defaultIsBad.html for more info